src/app/hooks/useOptimizelyScrollDepth/index.jsx

```diff
@@ -21,7 +21,7 @@ const useOptimizelyScrollDepth = () => {
   const [scrollSeventyFive, setScrollSeventyFive] = useState(false);
   const [scrollHundred, setScrollHundred] = useState(false);
 
-  const promoVariation = useOptimizelyVariation(OPTIMIZELY_CONFIG.featureId);
+  const promoVariation = useOptimizelyVariation()(OPTIMIZELY_CONFIG.featureId);
```

Comment:
Can we change the name of `promoVariation`? Think this was left over from an experiment that hardcoded this value, I think just `variation` or `experimentVariation` will suffice